DOCKER/Dockerfile

```diff
@@ -15,9 +16,34 @@ RUN apt-get update && \
   rm -rf /var/lib/apt/lists/*
 
 # set the repo and install tendermint
-ENV repo /go/src/github.com/eris-ltd/eris-db
-ADD . $repo
-WORKDIR $repo
-RUN cd ./cmd/erisdb && go install
-USER eris
-ENTRYPOINT ["erisdb"]
+WORKDIR $GOPATH/src/
+ENV REPO github.com/tendermint/tendermint
+ENV BRANCH permissions
+RUN mkdir --parents $REPO
+WORKDIR $GOPATH/src/$REPO
+RUN git clone https://$REPO . && \
+  git checkout $BRANCH && \
+  make && \
+  mv ./build/* /usr/local/bin/
+
+# set the repo and install erisdb
+ENV REPO /go/src/github.com/eris-ltd/eris-db
+COPY DOCKER/start.sh /usr/local/bin/erisdb-wrapper
+COPY . $REPO
+WORKDIR $REPO
+RUN cd ./cmd/erisdb && \
+  go build && \
+  mv erisdb /usr/local/bin/ && \
+  cd ../erisdbss && \
+  go build && \
+  mv erisdbss /usr/local/bin/
+
+# persist data, set user
+VOLUME /home/$USER/.eris
+WORKDIR /home/$USER/.eris
+USER $USER
+RUN mkdir --parents /home/$USER/.eris/blockchains/tendermint
+ENV TMROOT /home/$USER/.eris/blockchains/tendermint
```

Comment:
this would be fine if we only have one blockchain in a container at a time, but what if we want to share a data container with the entire ~/.eris across service and chain containers?  then this will have to be something like blockchains/chainName/chainNumber, and we cant set it in the Dockerfile